### ISSUE TITLE
TypeError for page after redirect

### ISSUE THREAD

**Issue by bartdigitalpulse** [putyourlightson/craft-blitz#519]
Everytime we visit a url that is redirected by the Retour plugin we get an TypeError. Stack trace below.
Example redirect: https://mywebsite.com/nl/category/entry-slug-2 (no longer exists) to https://mywebsite.com/nl/entry-slug-2

The "category/entry-slug-2" does not exist in the cache folder. "category/entry-slug" exists. Tried to refresh cache but it doesn't work. 

PHP version	8.1.20
Database driver & version	MySQL 8.0.33
Craft edition & version	Craft Pro 4.4.5
Blitz	4.4.4

```
TypeError: putyourlightson\blitz\services\GenerateCacheService::save(): Argument #1 ($content) must be of type string, null given, called in /vendor/putyourlightson/craft-blitz/src/services/CacheRequestService.php on line 413
#8 /vendor/putyourlightson/craft-blitz/src/services/GenerateCacheService.php(397): putyourlightson\blitz\services\GenerateCacheService::save
#7 /vendor/putyourlightson/craft-blitz/src/services/CacheRequestService.php(413): putyourlightson\blitz\services\CacheRequestService::saveAndPrepareResponse
#6 /vendor/putyourlightson/craft-blitz/src/Blitz.php(307): putyourlightson\blitz\Blitz::putyourlightson\blitz\{closure}
#5 [internal](0): call_user_func
#4 /vendor/yiisoft/yii2/base/Event.php(312): yii\base\Event::trigger
#3 /vendor/yiisoft/yii2/base/Component.php(642): yii\base\Component::trigger
#2 /vendor/yiisoft/yii2/web/Response.php(340): yii\web\Response::send
#1 /vendor/yiisoft/yii2/base/Application.php(390): yii\base\Application::run
#0 /index.php(12): null
```

**Comment by bencroker** [comment]
Thanks for reporting this! Fixed in https://github.com/putyourlightson/craft-blitz/commit/04fdcd4c0396fedce68457e412049a82596af707 for the next release.

**Comment by bencroker** [comment]
Released in version [4.4.7](https://github.com/putyourlightson/craft-blitz/releases/tag/4.4.7).
